pvlib/iotools/pvgis.py

```diff
@@ -424,6 +448,12 @@ def get_pvgis_tmy(latitude, longitude, outputformat='json', usehorizon=True,
         base url of PVGIS API, append ``tmy`` to get TMY endpoint
     timeout : int, default 30
         time in seconds to wait for server response before timeout
+    utc_offset: int, default 0
+        Use to specify a timezone other than the default UTC zero. Will force
+        year to ``coerce_year`` if not zero.
+    coerce_year: int, default 1990
+        Use to force indices to desired year. Ignored if ``utc_offset`` is
+        zero.
```

Comment:
```suggestion
        Used to force indices to 1990. Set to True if ``utc_offset`` is
        non-zero.
```

Reply:
this doesn't follow the usage in other functions, and I think that pvgis main issue is the `utc_offset` so my preference is to keep both args and allow user to coerce years like other `iotools` do